src/simplicial-hott/09-yoneda.rzk.md

```diff
@@ -432,29 +429,65 @@ We prove the Yoneda embedding.
       ( is-segal-A)
       ( a)
       ( hom A a')
-      ( is-covariant-representable-is-segal
-        ( A)
-        ( is-segal-A)
-        ( a')) φ
+      ( is-covariant-representable-is-segal A is-segal-A a') φ
+
+#def htpy-yoneda-phi uses (funext)
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( a a' : A)
+  ( φ : ( x : A) → ( hom A a x → hom A a' x) )
+  : (x : A) → ( ( yoneda-embedding A is-segal-A a a' ) ( ( evid A a ( hom A a')) φ )) x = φ x
```

Comment:
```suggestion
  : (x : A) →
    yoneda-embedding A is-segal-A a a' (evid A a (hom A a') φ) x = φ x
```
There's a couple of things to point out here:
- Our conventions only stipulate placing spaces after opening parentheses that appear at the start of a line. The reason for that is how this makes your code nicely align with other indented subexpressions.
- Overly parenthesizing expressions is detrimental to readability. The general exceptions to this is when your expressions involve lambdas or commas, which can generally be confusing to parse without the added parentheses.
- We try to keep code within an 80-character line length limit.

Reply:
Also, there is no need to insert a space before a closing parenthesis.